shell/sct_run_batch.sh

```diff
@@ -52,27 +52,69 @@ command_open() {
     OPEN_CMD="xdg-open"
   fi
 }
+# Check if GNU parallel if the flag -p is used
+check_GNU_parallel() {
+  if [ -x "$(command -v parallel)" ]; then
+    echo 'GNU parallel is installed! Processing subjects in parallel using multiple cores.' >&2
+    par='true'
+  else
+    echo 'GNU parallel is not installed.' >&2
+    par='false'
+    while true; do
+        read -p "Do you wish to continue processing subjects sequentially [y/n]?" yn
+        case $yn in
+            [Yy]* ) break;;
+            [Nn]* ) exit;;
+            * ) echo "Please answer yes or no.";;
+        esac
+    done
+  fi
+}
+
+# Usage of this script
+usage() {
+  echo -e "Correct usage ./sct_run_batch.sh <parameters.sh> <process_data.sh> [-p]"
+  echo -e "-p  Uses GNU-parallel if installed"
+  echo -e "-s  processing subjects sequentially"
```

Comment:
it's getting confusing: why two flags p and s? if parallel is used, it necessarily means that subjects are **not** processed sequentially. Why not simply keeping one flag: `-p` which can take values 0 or 1 (1: default)?

Reply:
alternatively, you can get rid of -p and only keep -s